index.html

```diff
@@ -17,6 +17,77 @@
+           (which also contains the hares) of the order Lagomorpha (which also contains the pikas). 
+           Oryctolagus cuniculus includes the European rabbit species and its descendants, the world's 305 breeds[1] of domestic rabbit.
+            Sylvilagus includes 13 wild rabbit species, among them the seven types of cottontail. 
+            The European rabbit, which has been introduced on every continent except Antarctica, 
+            is familiar throughout the world as a wild prey animal and as a domesticated form of livestock and pet.
+             With its widespread effect on ecologies and cultures, the rabbit is, in many areas of the world,
+              a part of daily life—as food, clothing, a companion, and a source of artistic inspiration.</p>
+
+          
+          <div class="links">
+            <a href="https://www.google.com/search?q=why+art+is+important&oq=why+art+is+important&aqs=chrome..69i57j0i512l9.10168j0j9&sourceid=chrome&ie=UTF-8">Learn more</a>
+          </div>
+      </div>
+
+        <div class="frame">
+
+        <iframe width="695" height="391" src="https://youtu.be/XF4TimSeGiU" 
+        title="Thing you need to know about rabbits" frameborder="0" 
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      
+      </div>
+         
+<br>
+<div class="hello">
+  <ul class="arty2">Fun Facts About Rabbits</ul>
+
+  
+<Li>To express happiness, bunnies will sometimes jump around and flick their heads and feet. ...</li>
+<li>Like deer, a female rabbit is called a “doe” and a male rabbit is called a “buck.”</li>
+<li> bunny's big ears aren't just for listening! ...</li>
+<li> those amazing ears can also be rotated almost a full circle to 270 degrees.</li> 
+</div>
+<div class="learn">
+  <a href="https://www.google.com/search?q=rabbit&sxsrf=ALiCzsbWiomUQgyVRNn5zi_jWh7G11karw%3A1666496921113&source=hp&ei=mblUY6LRBOqdhbIPj7ydsAI&iflsig=AJiK0e8AAAAAY1THqfcu_qflUJu1IJpVpPjiK-2AmQbr&ved=0ahUKEwjigYDouPX6AhXqTkEAHQ9eByYQ4dUDCAo&uact=5&oq=rabbit&gs_lcp=Cgdnd3Mtd2l6EAMyBQgAEJECMgcIABBDEIsDMgcIABBDEIsDMgcIABCxAxBDMgQIABBDMgsILhCABBDHARCvATIICAAQgAQQsQMyCwgAEIAEELEDEMkDMhQILhCxAxDHARCvARCLAxCoAxCmAzILCC4QgAQQsQMQ1AI6BwgjEOoCECc6BAgjECc6EQguEIAEELEDEIMBEMcBENEDOg4ILhCABBCxAxCDARDUAjoLCC4QgAQQsQMQgwE6DgguEIAEELEDEMcBENEDOhQILhCABBCxAxCDARCLAxCbAxCoAzoICC4Q1AIQkQI6BAguEEM6CggAELEDEIMBEEM6CgguEMcBENEDEEM6EwguENQCEEMQiwMQqAMQmgMQmAM6CggAELEDEEMQiwM6CgguELEDEEMQiwM6BwguEEMQiwM6FAguEIAEELEDEIMBEMcBENEDEIsDOhQILhCABBCxAxCLAxCaAxCbAxCoAzoUCC4QsQMQxwEQrwEQiwMQpgMQqANQ2A1Y5hZg2hpoAXAAeACAAXuIAY8EkgEDNC4ymAEAoAEBsAEKuAED&sclient=gws-wiz">Learn more</a>
+
+</div>
+
+  <br>
+<div class="arty">
+  <h2 class=" links">Summary</h2>
+  <article>When you look at the rabbits they inspire us to look after our enviroment therefore we must protect this precious species <br>as they 
```

Comment:
nice use of an article here, although you need to add a couple more `article` tags, and use the pseudo selector `first child` to style the first one